linux_os/guide/system/software/updating/ensure_GPG_keys_are_configured/rule.yml

```diff
@@ -1,57 +1,53 @@
+{{% if product in ["sle12", "sle15"] %}}
+{{% set product_names = "SUSE Linux distributions" %}}
+{{% set pub_key_import = "$ sudo rpm --import package-signing-key-5-12-6.pub"%}}
+{{% elif "ubuntu" in product %}}
+{{% set product_names = "Debian, Ubuntu OSes Linux distributions" %}}
+{{% set pub_key_import = "$ sudo gpg --import package-signing-key-5-12-6.pub"%}}
+{{% else %}}
+{{% set product_names = "RedHat,Fedora,CentOs Linux distributions" %}}
+{{% set pub_key_import = "$ sudo rpm --import package-signing-key-5-12-6.pub"%}}
+{{% endif %}}
+
+
 documentation_complete: true
 
 prodtype: sle12,sle15
 
 title: 'Ensure GPG keys are configured'
 
 description: |-
-    Most distributions use a package manager such as yum, apt, or zypper to install and 
-    update software packages. They use their own GPG keys directly or at least ship them
-    installed with the minimal system.  But third party packages signed by a different 
-    GPG key need manual handling. The operation system or installed application can be 
+    The operation system or installed application can be 
```

Comment:
Could you keep the lines length aligned to the Style Guide, please? It would also make it more readable.